tests/integration/devfile/utils/utils.go

```diff
@@ -585,3 +586,28 @@ func DeleteLocalConfig(args ...string) {
 	}
 	helper.MatchAllInOutput(output, expectedOutput)
 }
+
+// VerifyCatalogListComponent verifies component exists in both S2I Component list and Devfile Component list
+func VerifyCatalogListComponent(output string, componentName []string) error {
+	var data map[string]interface{}
+	listItems := []string{"devfileItems"}
+
+	err := json.Unmarshal([]byte(output), &data)
```

Comment:
keep error in the block scope, for example 

if err := json.Unmarshal([]byte(output), &data); err != nil {
    return err
}
```

Reply:
Just maintaining the contiguous pattern like everywhere you will see https://github.com/openshift/odo/blob/master/tests/helper/helper_generic.go#L82 and thats fine both are same and it will give you better glace view.